docs/developing-key4hep-software/SinglePackage.md

```diff
@@ -1,26 +1,32 @@
 # Developing a single package
 
-For quick changes of, for example, a single package, it's possible to compile
-the package using cmake (after having sourced the release or nightlies) and then
-export some environment variables manually so that our local version will be
-picked up instead of the one in cvmfs:
-
-``` bash
-export PATH=/path/to/install/bin:$PATH
-export LD_LIBRARY_PATH=/path/to/install/lib:/path/to/install/lib64:$LD_LIBRARY_PATH
-export ROOT_INCLUDE_PATH=/path/to/install/include:$ROOT_INCLUDE_PATH
-export PYTHONPATH=/path/to/install/python:$PYTHONPATH
+For quick modifications, such as updating a single package, you can source the stack from cvmfs, compile a local version of the selected package, and export specific environment variables to ensure the local package is used instead of the one from cvmfs.
+
+## With `k4_local_repo`
+
+`k4_local_repo` is a helper command to simplify setting up the environment for single package development. This command becomes available immediately after sourcing the environment. To use it, navigate to the root directory of the package you want to develop and execute the command:
+
+```bash
+k4_local_repo <install_location>
```

Comment:
```suggestion
k4_local_repo [<install_location>]
```

To indicate it's optional and defaults to `install`?

Reply:
It should be said it defaults to install somewhere, since I believe most people don't use the argument.